opentelemetry/proto/metrics/v1/gen.go

```diff
@@ -0,0 +1,119 @@
+// Copyright 2019, OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Notes: This code generates the Kind enum in metrics.proto.  To run this
+// code, execute:
+//
+//   go run gen.go > out.go && go run out.go
+//
+// The output of this program MUST be tested in an external module,
+// using the output of protoc.
+
+package main
+
+import (
+	"fmt"
+	"strconv"
+	"strings"
+)
+
+var (
+	cumulativeProps = []string{"Instantaneous", "Delta", "Cumulative"}
+	addingProps     = []string{"Adding", "Grouping"}
+	monoProps       = []string{"Monotonic", ""}
+	syncProps       = []string{"Synchronous", "Asynchronous"}
+)
+
+func main() {
+	fmt.Println(`package main
```

Comment:
Building the output program in totality and then passing through [`format.Source`](https://golang.org/pkg/go/format/#Source) would account for the style differences this output has.

Reply:
Fixed. (Thanks I was looking for that!)